packages/contentstack-migration/examples/05-Update-reference-entry-from-mapper.js

```diff
@@ -0,0 +1,138 @@
+const fs = require('fs');
+const chalk = require('chalk');
+module.exports = async ({ migration, stackSDKInstance, managementAPIClient, config }) => {
+  const modules = ['entries', 'assets', 'extensions'];
+
+  const readAllModulesUids = (filePath) => {
+    let uidMapping = [];
+    modules.forEach((module) => {
+      if (fs.existsSync(`${filePath}/mapper/${module}/uid-mapping.json`)) {
+        let mappedIds = fs.readFileSync(`${filePath}/mapper/${module}/uid-mapping.json`, 'utf-8');
+        mappedIds = JSON.parse(mappedIds);
+        uidMapping = { ...uidMapping, ...mappedIds };
+      } else {
+        console.log(chalk.red(`Mapper Does not exist for module '${module}'`));
```

Comment:
I think yes, as we will be mentioning the modules which this script will cover and in case any modules mapping is missing , we know, but still this script will run only after import if the there is no mapper for a module that means nothing got created and it's used nowhere in the entry. Let's remove it